src/StreamJsonRpc.Tests/DelimitedMessageHandlerTests.cs

```diff
@@ -152,18 +179,25 @@ private class DelayedWriter : DelimitedMessageHandler
     {
         internal readonly AsyncManualResetEvent WriteBlock = new AsyncManualResetEvent();
 
+        internal int WriteCoreCallCount;
+
         public DelayedWriter(Stream sendingStream, Stream receivingStream, Encoding encoding)
             : base(sendingStream, receivingStream, encoding)
         {
         }
 
+        internal bool WriteCoreAsyncCallsWriteBeforeReturningPublic { get; set; }
```

Comment:
Yes, it's because I expose what otherwise would only be `protected`. It's just a test project, so whether it's internal vs. public is immaterial. And adding "Internal" to the name doesn't IMO sound like I'm _expanding_ visibility. It's cuts either way, but I figured this was a bit more clearer to my intention.